package.json

```diff
@@ -15,6 +15,15 @@
   },
   "homepage": "https://github.com/CodeYourFuture/JavaScript-Core-2-Coursework-Week3-London8#readme",
   "devDependencies": {
-    "jest": "^27.4.7"
+    "@testing-library/dom": "^8.18.1",
+    "@testing-library/jest-dom": "^5.16.5",
+    "@testing-library/user-event": "^13.3.0",
```

Comment:
For @testing-library/user-event at version > 14, I get weird TypeError when requiring this module. Version 13.3.0 seems fine though :shrug: